Makefile

```
@@ -51,6 +51,26 @@ wake.db:	bin/wake bin/wakebox lib/wake/fuse-waked lib/wake/shim-wake
 install:	all
 	$(WAKE_ENV) ./bin/wake install $(DESTDIR)
 
+# Formats all .h and .cpp file under the current directory
+# It assumes clang is available on the PATH and will fail otherwise
+formatAll:
+	clang-format -i --style=file $(shell find . -not \( -path ./vendor -prune \) -type f \( -name "*.h" -o -name "*.cpp" -o -name "*.c" \))
+
+# Formats all changed or staged .h or .cpp files
+# It assumes clang is available on the PATH and will fail otherwise
+format:
+# Conditionally run clang-format based on the staged and changed files
+# || true is added after each if expression since they resolve with false when false
+# and we don't want make to report that as an error
+	@CHANGED_FILES=$$(git diff --name-only | grep '.h\|.cpp\|.c' | grep -v 'vendor/') && \
+	if [ ! -z $$CHANGED_FILES ]; then \
```

I think `! -z` is the same as `-n` which is also the no-option
```suggestion
	if [ $$CHANGED_FILES ]; then \
```

---

Ah right!